op-superchain/message.go

```diff
@@ -0,0 +1,30 @@
+package superchain
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type MessageSafetyLabel int
+
+const (
+	Invalid MessageSafetyLabel = iota
+	Safe
+	Finalized
+)
+
+type MessageIdentifier struct {
+	Origin      common.Address
+	BlockNumber uint64
+	LogIndex    uint64
+	Timestamp   uint64
+	ChainId     uint64
```

Comment:
chain id needs to be a u256 per the spec

Reply:
yep good callout, i'll refine this in a bit